backoffice/backoffice/users/urls.py

```diff
@@ -11,4 +12,5 @@
     path("~redirect/", view=user_redirect_view, name="redirect"),
     path("~update/", view=user_update_view, name="update"),
     path("<int:pk>/", view=user_detail_view, name="detail"),
+    path("login/success/", user_login_success, name="login_success"),
```

Comment:
@DonHaul  can we rename this as `/me` and @karolina-siemieniuk-morawska could integrate it in her PR. This will be similar to hep, user is logged in to orcid and UI check it's status from there.